jaxb-api/src/main/mr-jar/javax/xml/bind/ModuleUtil.java

```diff
@@ -158,7 +158,7 @@ static void delegateAddOpensToImplModule(Class[] classes, Class<?> factorySPI) t
                 continue;
             }
             //report error if they are not open to java.xml.bind
-            if (!classModule.isOpen(packageName, jaxbModule)) {
+            if (!classModule.isOpen(packageName, jaxbModule) && !"java.base".equals(classModule.getName())) {
```

Comment:
I think this should go few lines above to if (!classModule.isNamed()) .. to skip the rest.